front-packages/akeneo-design-system/src/components/Placeholder/Placeholder.tsx

```diff
@@ -1,36 +1,38 @@
-import React, {ReactElement} from 'react';
+import React, {cloneElement, FC, HTMLAttributes, ReactElement} from 'react';
 import styled from 'styled-components';
-import {getColor, getFontSize} from '../../theme';
+import {AkeneoThemedProps, getColor, getFontSize} from '../../theme';
 import {Override} from '../../shared';
 import {IllustrationProps} from '../../illustrations/IllustrationProps';
 
-const CenteredHelperContainer = styled.div`
-  text-align: center;
-  & > * {
-    display: block;
-    margin: auto;
-  }
+type PlaceholderSize = 'normal' | 'large';
```

Comment:
In button component we use the 'default' | 'small' can we used default ?
```suggestion
type PlaceholderSize = 'default' | 'large';
```

Reply:
👌